app/controllers/concerns/sufia/controller.rb

```diff
@@ -14,4 +16,15 @@ def current_ability
   def after_sign_in_path_for(_resource)
     sufia.dashboard_index_path
   end
+
+  # Ensure that the locale choice is persistent across requests
+  def default_url_options
+    { locale: I18n.locale }
```

Comment:
Does this need to merge with `super` in case another module is trying to inject its own behavior?